hotel_reservation/models/hotel_folio.py

```diff
@@ -0,0 +1,94 @@
+# See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class HotelFolio(models.Model):
+
+    _inherit = "hotel.folio"
+    _order = "reservation_id desc"
+
+    reservation_id = fields.Many2one(
+        "hotel.reservation", "Reservation", ondelete="restrict"
+    )
+
+    def write(self, vals):
+        res = super(HotelFolio, self).write(vals)
```

Comment:
I am unable to understand the need of this code. @vimalpatelserpentcs can you identify?